site/includes/deptfeatures-wet-126.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="gc-prtts mrgn-tp-md">
```

Comment:
This pattern should be contained in the template, not as an include

Reply:
Tu trouveras le code de la feature ici: https://wet-boew.github.io/themes-dist/GCWeb/component/gc-features-en.html